src/Okta.Sdk.UnitTests/Okta.Sdk.UnitTests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>    
-    <TargetFrameworks>netcoreapp2.0;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Why are we removing this?

Reply:
removed netcoreapp2.0 because it wasn't in the travis build environment.